mkosi.profiles/desktop/mkosi.conf.d/fedora/mkosi.conf.d/mkosi.conf

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=fedora
+Architecture=|x86
```

Comment:
Might as well drop this until we actually decide to support 32-bit (which I guess we never will)